core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/Tenant.java

```diff
@@ -32,8 +35,10 @@
 
     private String id;
 
+    @JsonInclude(Include.NON_EMPTY)
     private List<AggregationTemplate> aggregationTemplates = new ArrayList<>();
```

Comment:
> This PR is just the first part of a larger effort, but I want to get it
> merged as soon as possible because it gets out of control. The goal is

Agreed.

> to have Jackson library part of the core-impl, while core-api be just
> interfaces.

This goal is precisely what's being questioned :)

Nobody wants core-api alone. (By the way, now that we chose to support 
C* only, I'm not sure why it's still needed.)